RealmWeaver/ModuleWeaver.cs

```diff
@@ -810,6 +809,83 @@ private static bool IsRealmList(PropertyDefinition property)
         return property.PropertyType.Name == "RealmList`1" && property.PropertyType.Namespace == "Realms";
     }
 
+    private static bool IsDateTimeOffset(PropertyDefinition property)
```

Comment:
Yes it would, but I didn't want to diverge too much from the coding style. Also, when we add StyleCop, we'll probably have to reorder/reformat the whole file and it'll complain about constants/nameof/typeof :)